### PR TITLE
Highlight active link in side nav

### PR DESCRIPTION
**Pull request by keanulee**
image

### CODE REVIEW

File: docs/_layouts/post.html [hunk 1]
```diff
@@ -7,11 +7,11 @@
   <nav class="side-nav">
     <button id="toggleNavButton">Menu</button>
     <ul>
-      <li>
+      <li class="{% if page.url == '/guide.html' %}active{% endif %}">
```
Comment (straversi) [inline docs/_layouts/post.html]: If we add `permalink: /guide/` to the guide.md front matter, we could use the actual `/guide/` url here.

File: docs/css/main.css [hunk 2]
```diff
@@ -367,6 +367,10 @@ section p {
   line-height: 22px;
 }
 
+.side-nav li.active {
+  font-weight: bold;
```
Comment (straversi) [inline docs/css/main.css]: @yuinchien suggests using the current red for active, and `#222` for the rest.

**Comment by arthurevans** [comment]
I like it. I think to @straversi 's point, the bold black competes for attention with the body text, but will leave that for @yuinchien to cogitate on.

**Comment by yuinchien** [comment]
Looks good to me!

**Comment by arthurevans** [comment]
@keanulee Can you make a version of this PR for lit-element?

**Comment by keanulee** [comment]
@arthurevans can you merge https://github.com/Polymer/lit-html/pull/769 as well so I can update the other sites in one go?

**Pull request merged**